README.md

```diff
@@ -26,13 +26,15 @@
 
 **Goal:** Confirm proposal feasibility with Astro Maintainers and the [Technical Steering Committee (TSC)](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc).
 
-**Requirements:** An existing proposal (Stage 1). In addition, a proposal is more likely to be accepted if it is detailed and well thought-out, can demonstrate community interest, has at least one champion volunteer, and has buy-in/interest from Astro maintainer(s).
+**Requirements:** An existing proposal (Stage 1). In addition, a proposal is more likely to be accepted if it is detailed and well thought-out, can demonstrate community interest, has at least one champion volunteer, has a maintainer acting as a champion, and has buy-in/interest from Astro maintainer(s).
```

Comment:
It could be a nitpick, but here we use the term maintainer (lower case), but it's a generic term and a bit foggy. Who's an Astro maintainer? Did we want to use the L2 level from the governance?

Reply:
Yes, I did mean L2 maintainer. Should I put that in here?

Reply:
I think so, at least the first time we use the word